### ISSUE TITLE
Unable to find file Given As CLI Argument

### ISSUE THREAD

**Issue by jayrajput** [vajahath/open-swagger-ui#7]
I get following error. How to go about fixing/debugging it? I tried test.json, .\test.json and ./test.json, but none of them works.

```
C:\Users\XXXX\Desktop\Specification>open-swagger-ui .\test.json
× Loading file ..
Error: The given swagger file (C:/Users/jarajput/Desktop/Specification/test.json) could not be found.
    at C:\Users\jarajput\AppData\Roaming\npm\node_modules\open-swagger-ui\dist\app.js:126:27
    at step (C:\Users\jarajput\AppData\Roaming\npm\node_modules\open-swagger-ui\dist\app.js:33:23)
    at Object.throw (C:\Users\jarajput\AppData\Roaming\npm\node_modules\open-swagger-ui\dist\app.js:14:53)
    at rejected (C:\Users\jarajput\AppData\Roaming\npm\node_modules\open-swagger-ui\dist\app.js:6:65)
```

**Comment by vajahath** [comment]
Could you please take a look into this? I'm a bit tight with my schedules.
Since you have a Windows PC, you can try debugging this right away. It is simple code.
This might be the place which causing the issue: https://github.com/vajahath/open-swagger-ui/blob/79d337974cff2712f86db7077269db5a165c49de/src/app.ts#L50
Can you please take a look?
PR is most welcome. I'm happy to help.

**Comment by UlysseGuyon** [comment]
In the `getSwaggerDoc` function, the first action is to try to load the local file with `require`. At this stage, the function continues to try to find a way to load it, if the file does not exist but also if the file is not a well formed json. So you can get the error "file not found" even if the file was found but not well formed.

**Comment by vajahath** [comment]
Thanks @UlysseGuyon for your investigation. I've updated the code.
Could you please check the latest master in your env to see if the issue exists?

```sh
git clone
git checkout master
node -v
# >=10

npm install
npm run build
node dist/bin/open-swagger-ui.js <file/url> --open
```
Can you please add your findings here?
(@jayrajput )

**Comment by UlysseGuyon** [comment]
I tried with the last update and got : 

* For a malformed JSON file : `SyntaxError: Unexpected token � in JSON at position 0` and `{ details: 'Unable to parse the JSON swagger file. The JSON may be malformed. Try https://jsonlint.com/ to validate JSON' }`
* For non-existing file : `Error: The given swagger file ({"type":"path","path":"<MY_LOCAL_PATH>"}) could not be found.`
* For a good JSON file : `√ Loading file` and `Swagger open on port 5993`

So concerning my files, it seems that the problem has been fixed. 👍
